test/epoch_token_locker.js

```diff
@@ -72,12 +72,12 @@ contract("EpochTokenLocker", async (accounts) => {
       assert.equal((await epochTokenLocker.getPendingWithdrawBatchNumber(user_1, ERC20.address)).toNumber(), currentStateIndex.toNumber())
     })
   })
-  describe("withdraws", () => {  
+  describe("withdraws", () => {
```

Comment:
is this meant to be claim withdraw?

Reply:
yeah it is. Won't touch it in this PR though